src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write your code here
+function handleSuccess() {
+  const ressolveDiv = document.createElement('div');
+
+  ressolveDiv.className = 'message';
+  ressolveDiv.textContent = 'Promise was resolved!';
```

Comment:
Correct the variable name `ressolveDiv` to `resolveDiv` to ensure consistency and avoid errors.